Sources/Orbit/Components/Text.swift

```diff
@@ -106,8 +107,7 @@ public struct Text: View, FormattedTextBuildable {
         if #available(iOS 16.0, *), let isBold {
             return text
                 .bold(isBold)
-        }
-        else if let isBold, isBold {
+        } else if let isBold, isBold {
```

Comment:
`} else if isBold == true {` directly? 🤔

Also, aren't we piling up unnecessary attributes (`Text(...).bold(false).italic(false)`) or it is built up for a reason to maybe override it in the underlying hierarchy? Not sure whether this is applicable on some concatenation atm.

Reply:
I'm going with "not my problem" on this one... I just saw suboptimal formatting 😅

Reply:
Legit I guess 😄👌